src/fibaroAccessory.ts

```diff
@@ -181,7 +181,6 @@ export class FibaroAccessory {
       case 'com.fibaro.rollerShutter':
       case 'com.fibaro.FGWR111':
       case 'com.fibaro.remoteBaseShutter':
-      case 'com.fibaro.barrier':
       case 'com.fibaro.baseShutter': // only if favoritePositionsNativeSupport is true otherwise it's a garage door
```

Comment:
case 'com.fibaro.baseShutter': 
is present 2 times

Reply:
YES - it is your code. It is present 2 times because first time it checks additional conditions- "// only if favoritePositionsNativeSupport is true otherwise it's a garage door" And if it does not meet them then device is as garage door.

Reply:
Yeah, but there was also a comment to bypass the linter:

// eslint-disable-next-line no-duplicate-case, no-fallthrough

Reply:
Sorry, I didn't know this comment serves it.